Source/WebKit/WebProcess/Plugins/PDF/PDFPlugin.h

```diff
@@ -86,7 +86,7 @@ class PDFPlugin final : public PDFPluginBase {
 
 
     void paintControlForLayerInContext(CALayer *, CGContextRef);
-    void setActiveAnnotation(RetainPtr<PDFAnnotation>&&) final;
+    void setActivePDFPluginAnnotation(RetainPtr<PDFAnnotation>&&) final;
```

Comment:
I'm not convinced by the naming; also `PDFPluginAnnotation` implies that it belongs to the `PDFPlugin` class which isn't true in the UnifiedPDF code.

Reply:
I'll modify the naming so that it better represents that this is a HTML annotation. I'll leave any renaming out of this patch and have a dedicated renaming patch since I'll want to modify the class names also